### ISSUE TITLE
Radio buttons don't support aria-labelledby when adding iterations

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#3383]
Following #241.

For `staticControl.isFull`, we must return the id of the element which is the group:

```html
<span class="xforms-items" role="radiogroup" aria-labelledby="" aria-describedby="">
```

Currently, that element doesn't have an id.

**Comment by ebruchez** [comment]
This appears to work now, possibly due to changes for #5367.
